sdcm/ec2_client.py

```diff
@@ -40,16 +39,16 @@ class CreateSpotFleetError(ClientError):
     pass
 
 
-class EC2Client():
+class EC2Client:
 
-    def __init__(self, timeout=REQUEST_TIMEOUT, region_name=None, spot_max_price_percentage=None):
+    def __init__(self, timeout=REQUEST_TIMEOUT, region_name=None, spot_max_price_percentage=1.0):
         self._client = self._get_ec2_client(region_name)
         self._resource = boto3.resource('ec2', region_name=region_name)
         self.region_name = region_name
         self._timeout = timeout  # request timeout in seconds
         self._price_index = 1.5
         self._wait_interval = 5  # seconds
-        self.spot_max_price_percentage = spot_max_price_percentage
+        self.spot_max_price_percentage = 1.0 if spot_max_price_percentage > 1.0 else spot_max_price_percentage
```

Comment:
why are you adding this logic here ? this parameter is coming from sct_config.py, it should be tested validated there.
if you want to check it between 0.0 - 1.0, please assert here. 
and the default should be 1.0 for it, default for it is defined in `defaults/default_aws.yaml`, since it's a configuration option, we keep those defaults in one place

Reply:
I used this object and got an error...
But,  if there is one place I can revert my changes.